lib/models/device.coffee

```diff
@@ -249,6 +249,41 @@ exports.getApplicationName = (uuid, callback) ->
 	exports.get(uuid).get('application_name').nodeify(callback)
 
 ###*
+# @summary Get application container information
+# @name getApplicationInfo
+# @public
+# @function
+# @memberof resin.models.device
+#
+# @param {String} uuid - device uuid
+# @fulfil {Object} - application info
+# @returns {Promise}
+#
+# @example
+# resin.models.device.getApplicationInfo('7cf02a6').then(function(appInfo) {
+# 	console.log(appInfo);
+# });
+#
+# @example
+# resin.models.device.getApplicationInfo('7cf02a6', function(error, appInfo) {
+# 	if (error) throw error;
+# 	console.log(appInfo);
+# });
+###
+exports.getApplicationInfo = (uuid, callback) ->
+	exports.get(uuid).then (device) ->
```

Comment:
It's a tricky one, because the idea was to work nicely with custom supervisor builds (which might report a non-semver version or whatever they want) and also have custom endpoints or different behaviour, so forwarding dumbly works the best with that.  The SDK however is linked to the official resin supervisor, as it expects the endpoints that exist to match our supervisor and to behave in the same way, so we already have a dependency on our supervisor in the SDK and so can use that existing requirement to add additional linking.